api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -99,6 +99,11 @@ message RateLimit {
   // Disabled by default.
   XRateLimitHeadersRFCVersion enable_x_ratelimit_headers = 8
       [(validate.rules).enum = {defined_only: true}];
+
+  // Disables emitting the :ref:`x-envoy-ratelimited<config_http_filters_router_filter_x-envoy-ratelimited>` header
+  // in case of 429 responses.
```

Comment:
Probably more descriptive to say "in case of rate limiting (i.e. 429 responses)."

Reply:
fixed